OnlineStore/data/src/main/java/com/marina/dao/CategoriesDAO.java

```diff
@@ -0,0 +1,22 @@
+package com.marina.dao;
+
+import com.marina.entities.Category;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class CategoriesDAO extends CommonDAO<Category> {
+
+    @Override
+    public void completeStatement(PreparedStatement statement, List<Category> list) {
```

Comment:
it is better to thow ex (e/g unssoported operation, or similar) or implement method. Just not leave him empty)